@here/harp-geoutils/lib/coordinates/GeoPolygon.ts

```diff
@@ -0,0 +1,203 @@
+/*
+ * Copyright (C) 2017-2020 HERE Europe B.V.
+ * Licensed under Apache 2.0, see full license in LICENSE
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import { Vector2 } from "three";
+
+import { GeoBox } from "./GeoBox";
+import { GeoCoordinates, MAX_LATITUDE, MIN_LATITUDE } from "./GeoCoordinates";
+import { GeoCoordinatesLike } from "./GeoCoordinatesLike";
+import { GeoPolygonLike } from "./GeoPolygonLike";
+
+type MinThreeItemsArray<T> = [T, T, T, ...T[]];
+
+export type GeoPolygonCoordinates = MinThreeItemsArray<GeoCoordinatesLike | GeoCoordinates>;
+
+/**
+ * A GeoPolygon in 2D Space (altitudes will be ignored).
+ * Coordinates are expected in counter-clockwise order, for concave polygons a sorting is
```

Comment:
for concave polygons?

Reply:
no, if they are already ordered also convex ones should work. just the sorting might fail